### DISCUSSION TITLE
Which *.apk do I need to use?

### DISCUSSION THREAD

**Discussion by themanfrommoon** [OpenDTU-App/opendtu-react-native#17] [Q&A] [answered]
Hi!

Great work!
Thank you!
I would like to test it.
I found 10 different *.apk
I can't find any description?!
What is the difference?
Which *.apk do I need to use?

Thank you and best regards,
Chris

**Comment by CommanderRedYT** [comment]
Hi, build process is kind of  messy right now. Depending on how old your phone is, you either need `arm64-v8a` (newer phones) or `armeabi-v7a` (older phones).  I edited https://github.com/OpenDTU-App/opendtu-react-native/releases/tag/v0.1.4 to only contain the correct files. If something does not work, feel free to ask again! 

I will also try to make the releases more clear, and remove the files that are not needed!

**Reply by CommanderRedYT** [reply, marked as answer]
I reworked part of the CI, it should be more clear now. There is also a universal apk which should work on all platforms!

**Reply by mirkokarst19288** [reply]
Hi. Gibt es die Möglichkeit mehr Daten anzeigen zu lassen in LiveDaten . Also die einzelnen Strings und so. Das finde ich , fehlt noch . Gesamt Ertrag,  einzelne String Module , watt ,volt usw......
Kann mann auch per prometheus mehr anzeigen lassen ? 

Mfg

**Reply by CommanderRedYT** [reply]
Hi, the app still is in beta. If you got further questions, please create new threads instead of using old ones that talk about something else.

**Comment by Schmalzried** [comment]
Hi, thx for your App!
For a Samsung Tablet 9, which has Android 4.4, what apk would I need?
Are there any prerequisites?

**Reply by CommanderRedYT** [reply]
Interesting, I do not know if the app works with that old Android version. I would recommend the universal APK, this should work on every type of android device. 

I have to already say tho, there might be the case that it will not work because of a higher minimum android API version.

It could also be that I'm not able to decrease the required API version. Then, the only way I see is installing something like lineage on your device. But let's test it before!

**Reply by Schmalzried** [reply]
Thx for your quick reply.
I tried OpenDTUApp-universal but I get a parse error.
Yes lineage is an option but I don't know yet whether my old tablet is supported.
Actually I am  an iPhone user, has anyone tried your app there?

**Reply by CommanderRedYT** [reply]
I did not configure anything from apple yet. Also I am not sure what exactly you mean by parse error
